### ISSUE TITLE
new-lg4ff not updated / not current

### ISSUE THREAD

**Issue by stefanpausch** [chunksin/WiPiNetbooter#7]
tl;dr: Wipi (with current Updates) does still use (very?) old version of new-lg4ff

Hello,

running latest version of wipi (updated by web interface). 

After the update an old version of new-lg4ff is still being used and "openffb -a" can not see the g29 / g25 wheels i am using. 

Updated new-lg4ff to 0.4.0 manually and "openffb -a" sees the wheels.

There is an /usr/src/new-lg4ff-0.3.0  directory on the wipi image. My guess it that there is a logic to update to that version, but there is a bug in the update script and new-lg4ff never gets updated.

steps to reproduce:

- write the current wipi image to an sdcard
- enable openffb
- update wipi to the current version 
- connect either a logitech G25 or G29 wheel
- login via ssh and run "openffb -a" ... You get an error message "No haptic devices found"
- get latest verson of new-lg4ff from https://github.com/berarma/new-lg4ff
- unzip all contents to /usr/src/new-lg4ff
- run: sudo dkms install /usr/src/new-lg4ff
- reboot
- run "openffb -a" again and suddenly the wheels are found

```bash
[  102.402434] logitech 0003:046D:C299.0005: input,hidraw0: USB HID v1.11 Joystick [G25 Racing Wheel] on usb-3f980000.usb-1.1.2/input0
[  102.402489] logitech 0003:046D:C299.0005: Force feedback support for Logitech Gaming Wheels

[  119.815720] logitech 0003:046D:C24F.0003: input,hidraw0: USB HID v1.11 Joystick [Logitech G29 Driving Force Racing Wheel] on usb-3f980000.usb-1.1.2/input0
[  119.816086] logitech 0003:046D:C24F.0003: Force feedback support for Logitech Gaming Wheels
```

dmesg of both G25 and G29 wheels after updating new-lg4ff to 0.4.0:

```bash
[  179.981124] logitech 0003:046D:C24F.0006: input,hidraw0: USB HID v1.11 Joystick [Logitech G29 Driving Force Racing Wheel] on usb-3f980000.usb-1.1.2/input0
[  179.981472] logitech 0003:046D:C24F.0006: Force feedback support for Logitech Gaming Wheels (0.4.0)

[  110.075695] logitech 0003:046D:C299.0004: input,hidraw0: USB HID v1.11 Joystick [G25 Racing Wheel] on usb-3f980000.usb-1.1.2/input0
[  110.075760] logitech 0003:046D:C299.0004: Force feedback support for Logitech Gaming Wheels (0.4.0)
```

openffb -a output after updatig to new-lg4ff to 0.4.0:

```bash
----------------------------------------------------
Available Devices:
----------------------------------------------------
Device[0] Simplified Name        : g25-racing-wheel
Device[0] Real Name              : G25 Racing Wheel
Device[0] Vendor/Product/Version : 046d/c299/0111
Device[0] Driver Version         : 1.0.1
Device[0] path                   : /dev/input/event0

----------------------------------------------------
Available Devices:
----------------------------------------------------
Device[0] Simplified Name        : logitech-g29-driving-force-racing-wheel
Device[0] Real Name              : Logitech G29 Driving Force Racing Wheel
Device[0] Vendor/Product/Version : 046d/c24f/0111
Device[0] Driver Version         : 1.0.1
Device[0] path                   : /dev/input/event0
```
